tests/remote_server_windows_test.go

```diff
@@ -0,0 +1,52 @@
+// +build windows
+
+package playwright_test
+
+import (
+	"bufio"
+	"log"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/mxschmitt/playwright-go"
+)
+
+type remoteServer struct {
+	url string
+	cmd *exec.Cmd
+}
+
+func newRemoteServer() *remoteServer {
+	driver, err := playwright.NewDriver(&playwright.RunOptions{})
+	if err != nil {
+		log.Fatalf("could not start Playwright: %v", err)
+	}
+	cmd := exec.Command(driver.DriverBinaryLocation, "launch-server", browserName)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		log.Fatalf("could not get stdout pipe: %v", err)
+	}
+	err = cmd.Start()
+	if err != nil {
+		log.Fatalf("could not start server: %v", err)
+	}
+	scanner := bufio.NewReader(stdout)
+	url, err := scanner.ReadString('\n')
+	url = strings.TrimRight(url, "\n")
+	if err != nil {
+		log.Fatalf("could not read url: %v", err)
+	}
+	return &remoteServer{
+		url: url,
+		cmd: cmd,
+	}
+}
+
+func (s *remoteServer) Close() {
```

Comment:
Ideally we have something like a "process launcher" which has this specific kill logic for linux/windows in place and not duplicate the whole remote server logic.